core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1962,6 +1962,20 @@ private[spark] object Utils extends Logging {
     path
   }
 
+  /**
+   * Updates Spark config with properties from a set of Properties.
+   * Provided properties have the highest priority.
+   */
+  def updateSparkConfigFromProperties(conf: SparkConf, properties: Map[String, String])
```

Comment:
If you're gonna break method declarations into multiple lines, follow the appropriate style:
https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Indentation